test/integration/features/addon-registry-route.feature

```diff
@@ -43,6 +43,20 @@ so that user can able to push their container images directly to registry and de
       And with up to "10" retries with wait period of "500ms" the "status code" of HTTP request to "/" of service "nginx" in namespace "myproject" is equal to "200"
       And with up to "10" retries with wait period of "500ms" the "body" of HTTP request to "/" of service "nginx" in namespace "myproject" contains "Welcome to nginx!"
 
+  Scenario: As a user, I can login to OpenShift integrated docker registry after restart minishift
+    Given Minishift has state "Running"
+     When executing "minishift stop" succeeds
+     Then executing "minishift start --iso-url centos" succeeds
```

Comment:
As communicated in irc please remove the flag ```--iso-url centos```. In line no 14 ```--iso-url centos``` has been unnoticing since we make centos as default iso. Please remove from both the places.